WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/AttributePickerViewController.swift

```diff
@@ -0,0 +1,151 @@
+        configureAttribute(cell: cell, attribute: variationModel.allAttributes[safe: indexPath.row])
+
+        return cell
+    }
+}
+
+// MARK: - UITableViewDelegate Conformance
+//
+extension AttributePickerViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+
+        presentAttributeOptions(for: variationModel.allAttributes[indexPath.row])
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return UITableView.automaticDimension
+    }
+
+    func tableView(_ tableView: UITableView, estimatedHeightForHeaderInSection section: Int) -> CGFloat {
+        return 44
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let headerID = TwoColumnSectionHeaderView.reuseIdentifier
+        guard let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: headerID) as? TwoColumnSectionHeaderView else {
+            assertionFailure("Could not find section header view for reuseIdentifier \(headerID)")
+            return nil
+        }
+
+        headerView.leftText = Localization.headerAttributes
+        headerView.rightText = nil
+
+        return headerView
+    }
+}
+
+// MARK: - Cell configuration
+//
+private extension AttributePickerViewController {
+    func configureAttribute(cell: TitleAndValueTableViewCell, attribute: ProductAttribute?) {
+        guard let attribute = attribute else {
+            return
+        }
+
+        let optionValue = variationModel.productVariation.attributes.first(where: { $0.id == attribute.attributeID && $0.name == attribute.name })?.option ??
+            Localization.anyAttributeOption
+
+        cell.updateUI(title: attribute.name, value: optionValue)
+        cell.accessoryType = .disclosureIndicator
+    }
+}
+
+// MARK: - Navigation actions handling
+//
+extension AttributePickerViewController {
+
+    @objc private func doneButtonPressed() {
+        // TODO-3518: Save updated attributes
+    }
+
+    private func presentAttributeOptions(for existingAttribute: ProductAttribute) {
+        // TODO-3515: Show options for attribute
+    }
+}
+
+private extension AttributePickerViewController {
+    enum Localization {
+        static let titleView = NSLocalizedString("Attributes", comment: "Edit Product Attributes screen navigation title")
+        static let doneNavBarButton = NSLocalizedString("Done", comment: "Done nav bar button title in Edit Product Attributes screen")
+        static let headerAttributes = NSLocalizedString("Options", comment: "Header of attributes section in Edit Product Attributes screen")
+        static let anyAttributeOption = NSLocalizedString("Any", comment: "Product varition attribute description where the attribute is set to any value.")
```

Comment:
tiny nit: There's a small typo in the comment (`Product varition` should be `Product variation`).

Reply:
Thanks! There is even similar comment in another file that I'll also update:

https://github.com/woocommerce/woocommerce-ios/blob/e69b54614f6dd6084eae35b0db203cc04b636090/WooCommerce/Classes/ViewRelated/Products/Edit%20Product/EditableProductVariationModel.swift#L226

Reply:
Fixed in 965724c